vms/platformvm/block/builder/helpers_test.go

```diff
@@ -76,9 +76,19 @@ var (
 	testSubnet1            *txs.Tx
 	testSubnet1ControlKeys = preFundedKeys[0:3]
 
+	// Node IDs of genesis validators. Initialized in init function
+	genesisNodeIDs []ids.NodeID
+
 	errMissing = errors.New("missing")
 )
 
+func init() {
+	for range preFundedKeys {
+		nodeID := ids.GenerateTestNodeID()
+		genesisNodeIDs = append(genesisNodeIDs, nodeID)
+	}
+}
```

Comment:
nit: we should pre-allocate this

```suggestion
func init() {
	genesisNodeIDs = make([]ids.NodeID, len(preFundedKeys))
	for i, _ := range preFundedKeys {
		genesisNodeIDs[i] = ids.GenerateTestNodeID()
	}
}
```

Reply:
Done